.travis.yml

```diff
@@ -21,9 +27,7 @@ env:
   - NODE_ENV=test DB=mysql IMG=aws
 
 before_install:
-  - npm install -g npm@2
```

Comment:
Actually, node 0.10 ships with with npm @ 1.4.x, which can fail to installl modules correctly at times. (Yeah, overwriting the npm@2 that comes with 0.12 and 4.x is lame, but npm@1 is lamer).